### ISSUE TITLE
Example project pathing and imports not working on all platforms

### ISSUE THREAD

**Issue by palewire** [california-civic-data-coalition/django-calaccess-raw-data#1160]
Mac OSX users report that executing manage.py commands from a virtualenv result in an import error.

```bash
$ python example/manage.py shell
```

And here's the kind of traceback we see:

```bash
Traceback (most recent call last):
  File "example/manage.py", line 9, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/gordo/.virtualenvs/django-calaccess-raw-data/lib/python2.7/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/Users/gordo/.virtualenvs/django-calaccess-raw-data/lib/python2.7/site-packages/django/core/management/__init__.py", line 302, in execute
    settings.INSTALLED_APPS
  File "/Users/gordo/.virtualenvs/django-calaccess-raw-data/lib/python2.7/site-packages/django/conf/__init__.py", line 55, in __getattr__
    self._setup(name)
  File "/Users/gordo/.virtualenvs/django-calaccess-raw-data/lib/python2.7/site-packages/django/conf/__init__.py", line 43, in _setup
    self._wrapped = Settings(settings_module)
  File "/Users/gordo/.virtualenvs/django-calaccess-raw-data/lib/python2.7/site-packages/django/conf/__init__.py", line 99, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/local/Cellar/python/2.7.8_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named settings
```

**Comment by palewire** [comment]
Other projects like django-debug-toolbar have moved to a flatter example project structure, which research by @gordonje suggests will solve this problem. However, if we change that directory tree, there will be a small cascade of other changes that are necessary. Here's my quick draft list of what all those would be.

- [ ] The remodel of the example directory to eliminate the project subdirectory and move all those files to the base
- [ ] Walk through the settings.py file and see if any of the settings need to be changed to reflect the new path. Right away, the ``BASE_DIR`` setting that on line 2 is one that screams out as possibly needing to change.
- [ ] Verify that the data and toolbox and test-data subdirectories are still properly discovered and work
- [ ] Update the TravisCI testing configuration file to the new settings paths
- [ ] Update our documentation for getting started to reflect the paths

**Comment by gordonje** [comment]
Gonna fix the user docs after we merge in #1180 because there might be some conflicts.

Didn't see anything that needed to change in MakeFile. The build jobs passed after pushing.

**Comment by gordonje** [comment]
Actually, not seeing any path issues in the user docs either. Gonna call this one done unless something else comes up.
